lib/averages.js

```diff
@@ -0,0 +1,43 @@
+function total(array){
+  let sum = 0;
+  array.forEach(function(value){
```

Comment:
love the use of `forEach` here. you could use `reduce` though to help with the summation!